mparticle/models/batch.py

```diff
@@ -102,7 +103,8 @@ def __init__(self, events=None, source_request_id=None, environment=None, ip=Non
             'consent_state': 'consent_state',
             'api_key': 'api_key',
             'mpid': 'mpid',
-            'mp_deviceid': 'mp_deviceid'
+            'mp_deviceid': 'mp_deviceid',
+            'context': 'context'
```

Comment:
Nitpick: trailing comma?
```suggestion
            'context': 'context',
```